fedimintd/src/lib.rs

```diff
@@ -59,6 +61,9 @@ pub fn attach_default_module_gen_params(
         .attach_config_gen_params(
             LEGACY_HARDCODED_INSTANCE_ID_LN,
             LightningGen::kind(),
-            LightningGenParams::regtest(bitcoin_rpc),
```

Comment:
How did this ever work?! :exploding_head:

Reply:
Nobody had run fedimint on anything except regtest since that commit